site/en/docs/web-platform/view-transitions/index.md

```diff
@@ -420,23 +416,23 @@ In the case where the thumbnail transitions to a larger image:
 
 The default transition is to cross-fade, which means the thumbnail could be cross-fading with a not-yet-loaded full image.
 
-One way to handle this is to wait for the full image to load before starting the transition. Ideally this would be done before calling `.start()`, so the page remains interactive, and a spinner can be shown to indicate to the user that things are loading. But in this case there's a better way:
+One way to handle this is to wait for the full image to load before starting the transition. Ideally this would be done before calling `.startViewTransition()`, so the page remains interactive, and a spinner can be shown to indicate to the user that things are loading. But in this case there's a better way:
 
 ```css
-::page-transition-outgoing-image(full-embed),
-::page-transition-incoming-image(full-embed) {
+::view-transition-old(full-embed),
+::view-transition-new(full-embed) {
   /* Prevent the default animation,
-  so both images remain opacity:1 throughout the transition */
+  so both views remain opacity:1 throughout the transition */
   animation: none;
   /* Use normal blending,
-  so the incoming image sits on top and obscures the outgoing image */
+  so the new view sits on top and obscures the old view */
   mix-blend-mode: normal;
 }
 ```
 
-Now the thumbnail doesn't fade away, it just sits underneath the full image. This means if the incoming image hasn't loaded, the thumbnail is visible throughout the transition. This means the transition can start straight away, and the full image can load in its own time.
+Now the thumbnail doesn't fade away, it just sits underneath the full image. This means if the new view hasn't loaded, the thumbnail is visible throughout the transition. This means the transition can start straight away, and the full image can load in its own time.
 
-This wouldn't work if the incoming image featured transparency, but in this case we know it doesn't, so we can make this optimization.
+This wouldn't work if the new view featured transparency, but in this case we know it doesn't, so we can make this optimization.
```

Comment:
Doesn't need to be in this edit but your example for keeping a video playing throughout the transition is also a good one to add here. This came up with 2 different partners.

Showing how it can be made seamless by keeping the DOM change instant + hiding the old image is a cool pattern!

Reply:
Agreed. I'll start making notes for things that should be added for the next pass of this article.